### ISSUE TITLE
A11y_ SQL Azure Data Studio _ Notebooks Python Dependencies Wizard _Screenreader: Screenreader is not announcing the information of the data grid which is below Install required kernel Dependencies.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4161]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yTCS;-BM_SQLAzureDataStudio_Oct2020; -SQL Azure Data Studio;;-FTP;-NVDA;-A11yMAS;-A11ySev2-DesktopApp;-Win10;-WCAG1.3.1;-Benchmark;

--- **Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup 
Windows Version: Windows10
NVDA Version 2019.3.1

--- **Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Press “Alt+Windows+N” keys to open new notebook
4. Open the Command Palette by pressing CTRL+SHIFT+P 
5. Type “Python” into the text box that appears. Select “Configure Python for Notebooks” with the help of down arrow key  from the menu. Python Dependencies Wizard will appear.
6. Navigate to edit button with the help of tab key, press enter and fill the details.
7. Navigate to next button with the help of tab key and press enter.
8. Step 2 Install Dependencies page will get open
9. Launch nvda and navigate to previous button with the help of tab key
10. Now with the help of left or right arrow key, try to access information of the data grid which is below Install required kernel Dependencies
11. Check whether Nvda announces data grid information or not

--- **Actual:**
Screenreader is not announcing the information of the data grid which is below Install required kernel Dependencies with the help of left or right arrow key.
**This issue is only observed with NVDA**
--- **Expected:**
Screenreader should announce the information of the data grid which is below Install required kernel Dependencies with the help of left or right arrow key.
--- **User Impact:**
Screenreader user will not get proper and complete information of the data grid which is below “Install required kernel Dependencies”.
--- **Attachment:**
